ocean/sediment/100m/tracer_regression/config_driver.xml

```diff
@@ -0,0 +1,8 @@
+<driver_script name="run_test.py">
+	<case name="init">
+		<step executable="./run.py" quiet="true" pre_message=" * Running init" post_message="     Complete"/>
+	</case>
+	<case name="forward">
+		<step executable="./run.py" quiet="true" pre_message=" * Running forward" post_message="     Complete"/>
+	</case>
+</driver_script>
```

Comment:
```suggestion
	<validation>
		<compare_fields file1="forward/output.nc">
			<field name="sedConcen" l1_norm="0.0" l2_norm="0.0" linf_norm="0.0"/>
			<field name="temperature" l1_norm="0.0" l2_norm="0.0" linf_norm="0.0"/>
			<field name="salinity" l1_norm="0.0" l2_norm="0.0" linf_norm="0.0"/>
			<field name="layerThickness" l1_norm="0.0" l2_norm="0.0" linf_norm="0.0"/>
			<field name="normalVelocity" l1_norm="0.0" l2_norm="0.0" linf_norm="0.0"/>
		</compare_fields>
	</validation>
</driver_script>
```
This is needed in order for this test to function as a regression test.  With these lines added, it will compare with the same file in a baseline run (if one is provided to `./manage_regression_suite.py`) and make sure all these fields are exactly (bit-for-bit) identical between the two runs.